### ISSUE TITLE
License

### ISSUE THREAD

**Issue by Excrulon** [Excrulon/Tree-of-Savior-Experience-Viewer#5]
Added GNU license just because it's best to keep it open+free. Any objections? Meant to do this from the start.

**Comment by laurencee** [comment]
Yeah GPL (GNU) or MIT are my preferred choices for open source work. 

GPL is generally a better choice for this sort of project since the main difference between MIT and the GPL is that a GPL license requires all derived work to also remain open source (can not be packaged and sold etc.).

**Comment by fxfighter-gaming** [comment]
Yeah GPL (GNU) or MIT are my preferred choices for open source work.

GPL is generally a better choice for this sort of project since the main difference between MIT and the GPL is that a GPL license requires all derived work to also remain open source (can not be packaged and sold etc.).
